server/vulnerabilities/nvd/cpe_matching_rule.go

```diff
@@ -0,0 +1,109 @@
+package nvd
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/Masterminds/semver"
+	"github.com/facebookincubator/nvdtools/wfn"
+)
+
+// CPEMatchingRuleSpec allows you to match againts a CPE. Version ranges are supported via SemVer constraints.
```

Comment:
typo againts

Reply:
Fixed in https://github.com/fleetdm/fleet/pull/12551/commits/efaf93dbc46664fce508d5abde02af145ff445c9